packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -474,7 +474,13 @@ function ViewGrid< Item >( {
 							'dataviews-no-results': ! isLoading,
 						} ) }
 					>
-						<p>{ isLoading ? <Spinner /> : empty }</p>
+						{ isLoading ? (
+							<p>
```

Comment:
Do we need "p" around the spinner?

Reply:
Without the `<p>` it _can_ cause a jump in the dataview content when the view flips from the loading state to the empty state, because the vertical margin from the `<p>` appears all of a sudden.

I'm being super extra paranoid about this particular case because the default spinner and empty message elements are what appear in the Gutenberg editor. And I definitely do not want to introduce visual changes to the Gutenberg editor while working on this standalone component.